libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp

```diff
@@ -244,10 +254,15 @@ bool AP_RangeFinder_LightWareI2C::sf20_init()
     // version strings for reporting
     char version[15] {};
 
-    sf20_get_version("?P\r\n", "p:", version);
-
-    if (version[0]) {
-        hal.console->printf("SF20 Lidar version %s\n", version);
+    // Attempt to get lidar version up to MAX_VERSION_STRING_RETRY_ATTEMPTS times. This allows the UART to flush in between inits.
+    memset(version, 0, sizeof(version));
+    for (int i = 0; i < MAX_VERSION_STRING_RETRY_ATTEMPTS; ++i) {
+        if (sf20_get_version("?P\r\n", "p:", version)) {
+            break;
+        }
+        else {
+            hal.scheduler->delay(10);
```

Comment:
Could you please define a static const for the delay time and add units to the name? For readability

Reply:
Done!